src/families/bitcoin/wallet-btc/crypto/bip32.ts

```diff
@@ -28,7 +29,13 @@ class BIP32 {
     const I = createHmac("sha512", this.chainCode).update(data).digest();
     const IL = I.slice(0, 32);
     const IR = I.slice(32);
-    const Ki = Buffer.from(publicKeyTweakAdd(this.publicKey, IL));
+    const Ki = Buffer.from(
+      typeof navigator !== "undefined" && navigator.product === "ReactNative"
```

Comment:
Just wondering, would it not be good to have the new code on all platforms? Desktop and mobile?

Reply:
no, the lib "secp256k1" is faster in LLD because of C++ but it is very slow in LLM as it takes js implementation as fallback plan：
so the lib "noble-secp256k1"(the new one added by me) is used by LLM only

Reply:
the way we "feature detect" ReactNative here is not really safe to me. it could be better to introduce a "implementation" concept and a "setImplementation" that the client side will have to call.

Reply:
this idea of having a separate setImplementation could come when we will really need it for when we inject `react-native-secp256k1` – meanwhile it's acceptable to take this quick win with the embedded branching – it could be good to add a comment on the line of code with:

```
// FIXME LIVE-1443
```

( see https://ledgerhq.atlassian.net/browse/LIVE-1443 )